venues/ICLR.cc/2018/Conference/python/reviewer-assignment-from-server.py

```diff
@@ -0,0 +1,89 @@
+'''
```

Comment:
why the name of the script is from-server?

Reply:
because i originally wrote another function that was supposed to do the assignment, but ICLR PCs decided to use TPMS instead. so i wanted to distinguish this one-time script from the others

Reply:
ok